CHANGELOG.md

```diff
@@ -47,6 +47,13 @@ INTERNAL:
 
 * Use Terraform `1.8.0-rc1` for bundling schemas ([#1669](https://github.com/hashicorp/terraform-ls/pull/1669))
 
+## 0.33.0-beta1 (29 February 2024)
+
+BUG FIXES:
+
+* Fix panics on `nil` expressions ([hcl-lang#376](https://github.com/hashicorp/hcl-lang/pull/376))
+* Fix panics on `nil` expressions in early decoder ([terraform-schema#324](https://github.com/hashicorp/terraform-schema/pull/324))
+
```

Comment:
This is occurring because we released a patch version of 0.32.8 while also releasing a 0.33 beta. This should only be because of this transition